### ISSUE TITLE
Problem in getDependentLibs of Crypto test

### ISSUE THREAD

**Issue by zzambers** [adoptium/aqa-tests#4537]
Running aqa-tests on our infra we see this problem:
```
...
getDependentLibs:
     [exec] --------------------------------------------
     [exec] path is set to /home/tester/aqa-tests/TKG/../TKG/lib
     [exec] task is set to default
     [exec] dependencyList is set to jtreg_6_1${openj9jtregtimeouthandler}
     [exec] --------------------------------------------
     [exec] Starting download third party dependent jars
     [exec] --------------------------------------------
     [exec] downloading dependent third party jars to /home/tester/aqa-tests/TKG/../TKG/lib
     [exec] downloaded dependent third party jars successfully

getJtreg:
    [mkdir] Created dir: /home/tester/aqa-tests/test-results/functional/security/Crypto
     [exec] gzip: jtreg_6_1.tar.gz: No such file or directory

BUILD FAILED
...
```

it seems there is some un-evaluated variable in dependencyList:
https://github.com/adoptium/aqa-tests/blob/440b4b1216bba2eedd7cdca975a3d28b12d51cd5/functional/security/Crypto/build.xml#L82
https://github.com/adoptium/TKG/blob/8ab5268b80c61487264de925d00fb165b5f2421f/scripts/getDependencies.pl#L51

error comes from Cryptotest wrapper, not sure why it only affects us though
(I think error probably also causes not downloading rest of dependencies for other tests...)

**Comment by zzambers** [comment]
@sophia-guo Not sure what is function of `openj9jtregtimeouthandler` variable, so I don't know what is correct fix here, or if this is config issue on our side.

**Comment by sophia-guo** [comment]
openj9jtregtimeouthandler is specific to openj9|ibm impl. Need to check why you have this issue.

**Comment by sophia-guo** [comment]
Yes, might need to add this check https://github.com/adoptium/aqa-tests/blob/440b4b1216bba2eedd7cdca975a3d28b12d51cd5/openjdk/build.xml#L72-L83. But curious why we didn't hit the issue in adoptium, checking now...

**Comment by zzambers** [comment]
I was also wondering, why this issue does not happen in adoptium runs... 
Thanks for looking into this.

Btw. job, where we see this problem has following BUILD_LIST:
`BUILD_LIST=system,perf,functional`

**Comment by sophia-guo** [comment]
Yes, in adoptium we do the pre-staged jars so all jars has been staged before running the tests.
https://github.com/adoptium/aqa-tests/blob/master/buildenv/jenkins/JenkinsfileBase#L187-L195

**Comment by zzambers** [comment]
Ok, I'll make that change and try it.  Thanks

**Comment by zzambers** [comment]
Fix has been merged, closing.
